### PR TITLE
Periodic synchronization with central instance changes

### PR DESCRIPTION
**Pull request by 47Balboa**

### CODE REVIEW

File: roda-core/roda-core/src/main/java/org/roda/core/common/SyncUtils.java [hunk 1]
```diff
@@ -382,4 +382,30 @@ public static void updateDistributedInstance(LocalInstance localInstance, Distri
     }
   }
 
+  public static Long getUpdatesFromDistributedInstance(LocalInstance localInstance) throws GenericException {
+    try {
+      AccessToken accessToken = TokenManager.getInstance().getAccessToken(localInstance);
+      String resource = RodaConstants.API_SEP + RodaConstants.API_REST_V1_DISTRIBUTED_INSTANCE + "updates"
```
Comment (006627) [inline roda-core/roda-core/src/main/java/org/roda/core/common/SyncUtils.java:388]: Create a constant for the text "update"

File: roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstance.java [hunk 2]
```diff
@@ -462,4 +470,36 @@ public static String removeSyncBundle(String bundleName, User user, String bundl
     return RODAInstanceHelper.removeSyncBundle(bundleName, bundleDirectory);
 
   }
+
+  public static Long retrieveUpdates(User user, String instanceIdentifier)
```
Comment (006627) [inline roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstance.java:474]: Change method name to "retrieveCentralInstanceUpdates"

File: roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstanceHelper.java [hunk 3]
```diff
@@ -126,6 +126,7 @@ public static Long synchronizeIfUpdated(User user) throws GenericException, NotF
     if (localInstance != null && SynchronizingStatus.ACTIVE.equals(localInstance.getStatus())) {
       Date fromDate = localInstance.getLastSynchronizationDate();
       Date toDate = new Date();
+      total+= SyncUtils.getUpdatesFromDistributedInstance(localInstance);
```
Comment (006627) [inline roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstanceHelper.java:129]: Please also change the name of this method "synchronizeIfUpdated" to "retrieveLocalInstanceUpdates"

**Review by 006627 — CHANGES_REQUESTED** [review]

File: roda-core/roda-core/src/main/java/org/roda/core/common/SyncUtils.java (hunk 1 above)
Comment (47Balboa) [reply roda-core/roda-core/src/main/java/org/roda/core/common/SyncUtils.java:388]: done

File: roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstance.java (hunk 2 above)
Comment (47Balboa) [reply roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstance.java:474]: done

File: roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstanceHelper.java (hunk 3 above)
Comment (47Balboa) [reply roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/RODAInstanceHelper.java:129]: done

**Review by 006627 — APPROVED** [review]

**Pull request merged**